### DISCUSSION TITLE
Installing Alpine UI from npm components

### DISCUSSION THREAD

**Discussion by acalvino4** [alpinejs/alpine#3340] [4. General]
Is there a way to install and use Alpine UI from npm rather than a cdn? I am able to `npm install -D alpinejs/ui`, but I'm guessing I should have something like `Alpine.initializeUIComponents()` in my entry js file, similar to how it's done for the plugins.

**Comment by SimoTod** [comment]
I suppose it's the same as all the other plugins.
```js
import Alpine from 'alpinejs'
import ui from '@alpinejs/ui'

Alpine.plugin(ui)
```

**Comment by ekwoka** [comment]
https://www.npmjs.com/package/@alpinejs/ui

It is, you just forgot the `@`
